src/Text/native/Text.js

```diff
@@ -1,37 +1,38 @@
+import React from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
-import { Platform } from 'react-native';
+import { Platform, Text } from 'react-native';
 import { responsiveSizePx } from '../../utils/reactNativeResponsiveSize';
 import theme from '../../theme/native';
 
 const isPlatformAndroid = Platform.OS === 'android';
 
 const fontFamilies = {
-  normal: isPlatformAndroid ? 'Roboto-Regular' : 'SFProDisplay-Regular',
-  medium: isPlatformAndroid ? 'Roboto-Medium' : 'SFProDisplay-Medium',
-  semibold: isPlatformAndroid ? 'Roboto-Medium' : 'SFProDisplay-Semibold',
-  bold: 'Averta-Bold',
+  normal: isPlatformAndroid ? 'RegularFontAndroid' : 'RegularFontIOS',
+  medium: isPlatformAndroid ? 'MediumFontAndroid' : 'MediumFontIOS',
+  semibold: isPlatformAndroid ? 'MediumFontAndroid' : 'SemiboldFontIOS',
+  bold: isPlatformAndroid ? 'BoldFontAndroid' : 'BoldFontIOS',
 };
 
-const Text = styled.Text`
+const CustomText = styled((props) => <Text {...props} />)`
```

Comment:
Should be `StyledText`

Reply:
alternatively you can also do 
`import { Platform, Text as ReactNativeText } from 'react-native';`
and then leave the styled component text as `Text` which IMO is more clear